src/multi-selector.js

```diff
@@ -146,8 +146,7 @@ export default class MultiSelector extends React.Component {
     });
   };
 
-  removeItem = (item, e) => {
-    e.stopPropagation();
```

Comment:
The stopPropagation is happening inside the Pill box component itself. Removing here to fix error with removing pills from the box.